site2/docs/reference-cli-tools.md

```diff
@@ -707,19 +707,19 @@ The table below lists the environment variables that you can use to configure th
 |BOOKIE_GC_LOG|Gc options to be passed to the jvm||
 
 
-### `auto-recovery`
-Runs an auto-recovery service daemon
+### `autorecovery`
+Runs an auto-recovery service
 
 Usage
 ```bash
-$ bookkeeper auto-recovery options
+$ bookkeeper autorecovery options
 ```
 
 Options
 
 |Flag|Description|Default|
 |---|---|---|
-|`-c`, `--conf`|Configuration for the auto-recovery daemon||
+|`-c`, `--conf`|Configuration for the autorecovery||
```

Comment:
`auto-recovery` (or `auto recovery`) is a correct word in the description, why modify that?

Reply:
I agree, we should keep the `auto-recovery` term.

Reply:
OK, I updated in https://github.com/apache/pulsar/pull/12989/commits/94ebed6125a68ce3c235b3e30abf7bbdf7bba3ea